policyengine_us/variables/gov/states/ca/tax/income/alternative_minimum_tax/ca_exemption_child_eligible.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ca_exemption_child_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible person for the California Minimum exemption amount"
+    documentation = (
+        "https://www.ftb.ca.gov/forms/2022/2022-540-p-instructions.html"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.CA
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.ca.tax.income.alternative_minimum_tax
+
+        head = person("is_tax_unit_head", period)
+        child = person("age", period) < p.age_threshold.lower
+        no_income = (person("age", period) == p.age_threshold.lower) & (
+            person("earned_income", period) == 0
```

Comment:
not sure whether a support variable needs to be added or created
![image](https://github.com/PolicyEngine/policyengine-us/assets/31402450/7a8c365a-7372-463e-be91-da405be329b1)